vcbuild.bat

```diff
@@ -498,6 +498,7 @@ echo   vcbuild.bat enable-vtune   : builds nodejs with Intel VTune profiling sup
 goto exit
 
 :exit
+exit /b %errorlevel%
```

Comment:
AFAIK `exit /b` just passes %ERRORLEVEL% by default...
[`EXIT without an ExitCode acts the same as goto:eof and will not alter the %ERRORLEVEL%`](https://ss64.com/nt/exit.html)